UnitySDK/Assets/ML-Agents/Scripts/Sensor/VectorSensor.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace MLAgents.Sensor
+{
+    public class VectorSensor : SensorBase
+    {
+        public List<float> observations;
+        int[] m_Shape;
+        string m_Name;
+
+        public VectorSensor(int observationSize, List<float> observations = null, string name = null)
+        {
+            if (name == null)
+            {
+                name = $"VectorSensor_size{observationSize}";
+            }
+
+            if (observations == null)
+            {
+                observations = new List<float>(observationSize);
```

Comment:
Won't use this for Agent right now, not sure if it's useful to be able to pass the List instead.